pkg/services/supportbundles/supportbundlesimpl/collectors.go

```diff
@@ -103,68 +104,79 @@ func settingsCollector(settings setting.Provider) supportbundles.Collector {
 	}
 }
 
-func pluginInfoCollector(pluginStore plugins.Store, pluginSettings pluginsettings.Service) supportbundles.Collector {
+func pluginInfoCollector(pluginStore plugins.Store, pluginSettings pluginsettings.Service, logger log.Logger) supportbundles.Collector {
 	return supportbundles.Collector{
 		UID:               "plugins",
 		DisplayName:       "Plugin information",
 		Description:       "Plugin information for the Grafana instance",
 		IncludedByDefault: false,
 		Default:           true,
 		Fn: func(ctx context.Context) (*supportbundles.SupportItem, error) {
-			type pluginInfo struct {
-				data  plugins.JSONData
```

Comment:
so instead of data we now parse information into different fields? 🤔

Reply:
`plugins.JSONData` contains way more information than what I think needs to be included in a support bundle (because it clutters up the data and could also be a security risk).

For context - when I first added this collector, it was during the hackathon, so it was a really rushed implementation. This PR attempts to clean up the code as well as the `plugins.json` generated as part of a support bundle.